src/cli/src/cluster/install/k8.rs

```diff
@@ -1,9 +1,34 @@
 use std::process::Command;
 use std::io::Error as IoError;
+use semver::Version;
 
 use super::*;
 
+fn pre_install_check() -> Result<(), CliError> {
+    let helm_version = Command::new("helm")
+        .arg("version")
+        .arg("--short")
+        .output()
+        .map_err(|err| {
+            CliError::Other(format!(
+                "Helm package manager not found: {}",
+                err.to_string()
+            ))
+        })?;
+
+    let version_text = String::from_utf8(helm_version.stdout).unwrap();
+    let version_text_trimmed = &version_text[1..].trim();
+    if Version::parse(&version_text_trimmed) <= Version::parse("3.2.0") {
+        println!(
```

Comment:
return this as error instead of printing out since pre-flight check failure is considered error in the beginning of the code